packages/mobile_app/lib/screens/daily/screen.dart

```diff
@@ -16,16 +18,25 @@ class DailyScreen extends StatelessWidget {
   DailyScreen({@required this.dt});
   @override
   Widget build(BuildContext context) {
-    final usersRepo = context.watch<UsersRepositoryFlutter>();
-    int currentDay = dtDay;
+    if (dt == dtDay)
+      return TodayScreen(dt: dt);
+    else
+      return _DailyScreen(dt: dt);
+  }
+}
 
-    UserPollBloc userPollBloc;
-    if (dt == currentDay) userPollBloc = context.bloc<UserPollBloc>();
-    print(dt);
-    print(currentDay);
+class TodayScreen extends StatelessWidget {
+  int dt;
+  TodayScreen({@required this.dt});
+  @override
+  Widget build(BuildContext context) {
+    final usersRepo = context.watch<UsersRepositoryFlutter>();
 
+    UserPollBloc userPollBloc = context.bloc<UserPollBloc>();
     ProphecyBloc prophet = context.bloc<ProphecyBloc>();
+
     prophet.add(CalculateProphecy(dt));
+    userPollBloc.add(UserPollRestartEvent());
```

Comment:
I dont like this piece of code.
Because it sends new events to bloc when widget is build.
And no matter if it rebuilded in purpose or because of some flutter glitch.
It will send new event that will be calculated.
And we cant go the other way in stateless widget.

In statefull we could do this mess in the init function.
But stateful widgets are slower.